http.go

```diff
@@ -45,7 +57,10 @@ func httpGetAddrs(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Header().Set("Transfer-Encoding", "chunked")
```

Comment:
Without a content length set, the chunked writer sets this header anyway as soon as response body writing starts via enc.Encode.  That is except for http/2 where chunked transfer encoding isn't a think anyway.